.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "typescript.tsdk": "./node_modules/typescript/lib"
+  "typescript.tsdk": "./node_modules/typescript/lib",
+  "prettier.enable": false
```

Comment:
Whoops, will remove this.

Reply:
You could keep it if you'd like. I think there's no harm having this.